src/engine/media/model.ts

```diff
@@ -0,0 +1,26 @@
+export type NIP98_event = {
+	status: string, 
+	message: string,
+    processing_url: string,
+	nip94_event : NIP94_event
+    }
+
+ export type NIP94_event = {
+    id : string,
+    pubkey: string,
+    created_at: string,
+    kind: 1063,
+    tags: [
+            ["url", string],
+            ["m", string],
+            ["x", string],
+            ["ox", string],
+            ["size", string],
+            ["dim",string],
+            ["magnet", string],
+            ["i", string],
+            ["blurhash", string]
+    ],
+    content: string,
+    sig : string,
+}
```

Comment:
No need to be so strict with typescript, I prefer to just use `Event` and assert stuff when needed

Reply:
removed